src/Codebreaker.WPF/Services/Navigation/PageServiceBuilder.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Codebreaker.WPF.Services.Navigation;
+
+internal class PageServiceBuilder
+{
+    private readonly Dictionary<string, Func<Page>> _pages = new();
+
+    private Func<Page>? _initialPage;
+
+    public PageServiceBuilder Configure<V>()
+        where V : Page, new() =>
```

Comment:
why use V for the generic type name? Use a T prefix, e.g. Dictionary<TKey, TValue>